### PR TITLE
Support multi-table sink feature for influxdb

### PR DESCRIPTION
**Pull request by 531651225**
### Purpose of this pull request
Support multi-table sink feature for influxdb
refer to https://github.com/apache/seatunnel/issues/5652

### Check list

* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [x] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by 531651225** [comment]
@hailin0 @EricJoy2048 please cc

**Comment by hailin0** [comment]
please fix ci

**Comment by hailin0** [comment]
cc @Hisoka-X

File: seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/sink/InfluxDBSinkFactory.java [hunk 1]
```diff
@@ -46,7 +56,7 @@ public String factoryIdentifier() {
     @Override
     public OptionRule optionRule() {
         return OptionRule.builder()
-                .required(URL, DATABASES, KEY_MEASUREMENT)
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/sink/InfluxDBSinkFactory.java:49]: please move `KEY_MEASUREMENT` to optional

Comment (531651225) [reply seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/sink/InfluxDBSinkFactory.java:49]: > please move `KEY_MEASUREMENT` to optional
Writing to config for a single table requires configuring KEY-MEASURE, but not for multi table writing. 
Reference kudu MultiTableSinkWriter
image

File: seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/sink/InfluxDBSinkFactory.java [hunk 2]
```diff
@@ -46,7 +56,7 @@ public String factoryIdentifier() {
...
+                .required(URL, DATABASES)
                 .bundled(USERNAME, PASSWORD)
                 .optional(
                         CONNECT_TIMEOUT_MS,
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/sink/InfluxDBSinkFactory.java:62]:
```suggestion
                        CONNECT_TIMEOUT_MS,
                        KEY_MEASUREMENT,
```

Comment (531651225) [reply seatunnel-connectors-v2/connector-influxdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/influxdb/sink/InfluxDBSinkFactory.java:62]: @Hisoka-X thinks, I fix it

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by hailin0 — APPROVED** [review]

**Pull request merged**